1_dynamic/future.k

```diff
@@ -228,7 +234,7 @@ module FUTURE
 /* Calls using closure desugar into future calls on the closure's object
 */
   rule <k>  methodClosure(Type,Class,OId,Ps,S) (Vs:Vals)
-        => future(F).get ...</k>
+        => future(F) ~> waitAndGet ...</k>
```

Comment:
De acord cu refolosirea sablonului, dar parca se intelege mai greu cu aceasta indirectare.
Cred ca ar deveni mai clar (cel putin din punctul meu de vedere) daca am face sablonul parametric (si strict), astfel incat sa avem `waitAndGet(future(F))`. 
In utilizarea precedenta am avea `O ! waitAndGet(Method(Vs))`,  care parca si ea s-ar citi mai usor.